Client/Layout/MainLayout.razor

```diff
@@ -2,15 +2,41 @@
 
 <RadzenComponents />
 <div class="page">
-    <div class="sidebar">
-        <NavMenu />
-    </div>
-
     <main>
-        <div class = "app_bar">
-            <AppBar/>
+        <div class="header">
+            <h1 class="title">
+                <img src="Logo.png" alt="logo" class="logo" />
+                Contrasting Horizons</h1>
+            <nav>
+                <ul class="nav_links">
+                    <li><NavLink href="/">Home</NavLink></li>
+                    <li><NavLink href="/Compare">Compare</NavLink></li>
+                    <li><NavLink href="/about">About</NavLink></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="search-container">
+            <input type="text" placeholder="Search for a country..." name="search" />
+            <button type="submit">Search</button>            
         </div>
 
+        <div>
+            <p class="paragraph_country">
```

Comment:
We probably not have this on all pages

Reply:
Its just a place holder for the country that the information would be shown. We can move it to a home page where the country of the day ond dashoboard is located. But at the moment we dont have any home page. Main is what we have. Maybe we can bring it up on the next sprint planning.

Reply:
Fixed it. Removed it from the MainLayout and instead put it in the Home page.